### ISSUE TITLE
Skins

### ISSUE THREAD

**Issue by CodingKuda** [nulcearc/agar-api-server#1]
Can you make a resporitry for the agario skins, so we can keep up with eachother. Thanks. Kuda

**Comment by nulcearc** [comment]
you mean repo where are all the skins in agario?

**Comment by CodingKuda** [comment]
@nulcearc Cause the new skins came out today. And yeah.

**Comment by nulcearc** [comment]
Yes i updated. Everything about Agario i'll keep in this Repository. I also want to find way to download all skin images from agario

**Comment by CodingKuda** [comment]
@nulcearc I got the troll skins, the 4th of july skins and the bear skins things

**Comment by CodingKuda** [comment]
@nulcearc  i can send them,

**Comment by nulcearc** [comment]
@CodingKuda you can find all the skins in agario here: https://configs-web.agario.miniclippt.com/live/v4/155/GameConfiguration.json.
 here are some png images i found :
https://configs-web.agario.miniclippt.com/live/v4/117/FunShine_Bear.png
https://configs-web.agario.miniclippt.com/live/v4/117/germany_hi.png

**Comment by CodingKuda** [comment]
these are mine 

https://configs-web.agario.miniclippt.com/live/v4/150/birthday_sanik.png
https://configs-web.agario.miniclippt.com/live/v4/150/birthday_cia.png
https://configs-web.agario.miniclippt.com/live/v4/150/birthday_lol.png
https://configs-web.agario.miniclippt.com/live/v4/150/birthday_troll.png
https://configs-web.agario.miniclippt.com/live/v4/150/birthday_doge.png

**Comment by nulcearc** [comment]
Cool! more:
https://configs-web.agario.miniclippt.com/live/v4/150/Bat.png
https://configs-web.agario.miniclippt.com/live/v4/150/Kraken.png
https://configs-web.agario.miniclippt.com/live/v4/150/Hunter.png
https://configs-web.agario.miniclippt.com/live/v4/150/T_Rex.png
https://configs-web.agario.miniclippt.com/live/v4/150/Raptor.png

**Comment by CodingKuda** [comment]
I need to new Food skins..can you find them @nulcearc

**Comment by nulcearc** [comment]
Here is one: https://configs-web.agario.miniclippt.com/live/v4/155/food_Chicken_Leg.png

**Comment by CodingKuda** [comment]
nice thanks. @nulcearc I will keep searching.

**Comment by CodingKuda** [comment]
@nulcearc wait, how did you find the name of the skin, when it wasn't in the https://configs-web.agario.miniclippt.com/live/v4/146/GameConfiguration.json. (game configuration) because i cant find it.

**Comment by CodingKuda** [comment]
@nulcearc i cant find any of the skins...

**Comment by CodingKuda** [comment]
@nulcearc you there?

**Comment by nulcearc** [comment]
@CodingKuda add me on skype: nuclear.agar

**Comment by CodingKuda** [comment]
@nulcearc  OK

**Comment by ItzLevvie** [comment]
"I also want to find way to download all skin images from agario"

This is how I done it with free skins:  https://github.com/ItzLevvie/Agario-skins/blob/master/freeSkins.sh

I'm going to plan on adding premium skins soon :)

**Comment by ItzLevvie** [comment]
I managed to get all of the premium skins except for  2 or 3 of them which I can't get :/

**Comment by nuclearc** [comment]
wow! is it for linux only?

**Comment by ItzLevvie** [comment]
"wow! is it for linux only?" @nuclearc

Yep.

"some skins have 'hi.png' at the end, some not :/ wtf is that?"

If you ran my script, and there was some "hi.png" then use my new commited version

{IMG}_hi.png = 512 x 512 pixels (This is what all of the skins supposted to be
{IMG}.png = 150 x 150 (I believe this is used for shops, not sure really)

**Comment by nuclearc** [comment]
@ItzLevvie i created my script too, now it parses GameConfig.json and downloads all premium skins
